test/integration/volume/attach_detach_test.go

```diff
@@ -100,6 +100,9 @@ func TestPodDeletionWithDswp(t *testing.T) {
 	pod := fakePodWithVol(namespaceName)
 	podStopCh := make(chan struct{})
 
+	go informers.Core().V1().PersistentVolumeClaims().Informer().Run(podStopCh)
```

Comment:
yes, it doesn't matter because there are no pv's or pvcs in this test. But I will make these use stopCh to isolate the usage of podStopCh more clearly.